packages/qgrid-ngx-plugins/src/lib/action-bar/action-bar.component.ts

```diff
@@ -25,6 +25,24 @@ export class ActionBarComponent implements OnInit {
 
 		observe(model.actionChanged)
 			.subscribe(e => {
+				const initialItems = e.state.items;
+				const sortedItems = initialItems.sort((a: Action, b: Action) => {
+					return a.command.priority - b.command.priority;
+				});
+
+				const sorted = sortedItems.every((item: Action, index: number) => {
+					return item.title = initialItems[index].title;
```

Comment:
= or === ?

Reply:
maybe a good idea to check if array is sorted first, and if not sort it out?

https://stackoverflow.com/questions/19458278/check-if-an-array-is-sorted-return-true-or-false